job-tutorial/README.md

```diff
@@ -138,19 +138,19 @@ Below we show a complete list of environment variables accessible in a Docker co
 | PAI_DATA_DIR                       | `dataDir` in config file                 |
 | PAI_OUTPUT_DIR                     | `outputDir`in config file or the generated path if `outputDir` is not specified |
 | PAI_CODE_DIR                       | `codeDir` in config file                 |
+| PAI_TASK_ROLE_LIST                 | Comma separated all task role names in config file |
 | PAI_TASK_ROLE_NAME                 | `taskRole.name` of current task role     |
-| PAI_TASK_ROLE_NUM                  | `taskRole.number` of current task role   |
+| PAI_TASK_ROLE_NUM                  | `taskRole.taskNumber` of current task role |
```

Comment:
It seemed task role num mean number of task roles. Shall you gives out a better name without this kind of confusion?

Reply:
Maybe PAI_TASK_NUM is enough?

Reply:
ok